shared-bindings/usb/__init__.c

```diff
@@ -30,15 +30,48 @@
 
 #include "shared-bindings/usb/__init__.h"
 #include "shared-bindings/usb/core/__init__.h"
+#include "supervisor/usb.h"
 
 //| """PyUSB-compatible USB host API
 //|
 //| The `usb` is a subset of PyUSB that allows you to communicate to USB devices.
 //| """
+//|
+//| def set_user_keymap(keymap: ReadableBuffer, /) -> None:
```

Comment:
Let's not add this to the `usb` module because it is meant to be a subset of pyusb. How about on `supervisor` or `usb_host`?

Reply:
I'll move it to `usb_host`.

Reply:
done